### PR TITLE
Add more PMD items

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

File: mods/pmd/moves.js [hunk 1]
```diff
@@ -558,4 +558,113 @@ exports.BattleMovedex = {
...
+		isNonstandard: true,
+		name: "Sleep Seed",
+		basePower: 0,
+		pp: 0.625,
+		priority: 0,
+		flags: {authentic: 1, bullet: 1, snatch: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, "Bullet Seed", target);
+			this.add('-anim', source, "Rest", target);
```
Comment (HoeenCoder) [inline mods/pmd/moves.js]: `target, "Rest", target` for proper anim

File: mods/pmd/moves.js [hunk 2]
```diff
@@ -558,4 +558,113 @@ exports.BattleMovedex = {
...
+		},
+		status: 'slp',
+		ignoreImmunity: true,
```
Comment (HoeenCoder) [inline mods/pmd/moves.js]: Sleep seeds dont ignore Immuities.

File: mods/pmd/moves.js [hunk 3]
```diff
@@ -558,4 +558,113 @@ exports.BattleMovedex = {
...
+		basePower: 0,
+		category: "Status",
+		isNonstandard: true,
+		pp: 0.625,
+		flags: {snatch: 1},
+		onPrepareHit: function (source, target) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, "Smokescreen", target);
+		},
+		boosts: {accuracy: -3},
```
Comment (HoeenCoder) [inline mods/pmd/moves.js]: this is way too much, `-1` will work well (evasion orb is +1)

**Review by HoeenCoder — CHANGES_REQUESTED** [review]

File: mods/pmd/moves.js (hunk 1 above)
Comment (DeathlyPlays) [reply mods/pmd/moves.js]: Ah I didn't code that part I'll fix it xD

File: mods/pmd/moves.js (hunk 2 above)
Comment (DeathlyPlays) [reply mods/pmd/moves.js]: Eh mk

File: mods/pmd/moves.js (hunk 3 above)
Comment (DeathlyPlays) [reply mods/pmd/moves.js]: Ok

**Review by HoeenCoder — APPROVED** [review]

**Pull request merged**